test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DacFx/DacFxServiceTests.cs

```diff
@@ -942,6 +975,24 @@ private void VerifyAndCleanup(string filePath)
             }
         }
 
+        private void VerifyContentAndCleanup(string baselineFilePath, string outputFilePath)
+        {
+            // Verify it was created
+            Assert.True(File.Exists(outputFilePath));
```

Comment:
Can you add a message to this one so the failure doesn't just say "expected true, got false"?

Reply:
Added a message here.